api/openapi_server/configs/production.py

```diff
@@ -0,0 +1,55 @@
+from dataclasses import dataclass
+import re
+
+from openapi_server.configs.huu_config import HUUConfig, secret_str_field
+
+@dataclass(frozen=True)
+class ProductionHUUConfig(HUUConfig):
+    ENV: str = "production"
+    FLASK_DEBUG: bool = False
+    TESTING: bool = False 
+    SECRET_KEY: str = secret_str_field()
+    DATABASE_URL: str = ""
+    ROOT_URL: str = "homeunite.us"
+    COGNITO_CLIENT_ID: str = secret_str_field()
+    COGNITO_CLIENT_SECRET: str = secret_str_field()
+    COGNITO_REGION: str = secret_str_field()
+    COGNITO_REDIRECT_URI: str = secret_str_field()
+    COGNITO_USER_POOL_ID: str = secret_str_field()
+    COGNITO_ACCESS_ID: str = secret_str_field()
+    COGNITO_ACCESS_KEY: str = secret_str_field()
+
+    def post_validate(self):
+        super().post_validate()
+        self.validate_secret_key(self.SECRET_KEY)
+        if (self.FLASK_DEBUG):
+            raise ValueError("Debug mode is not supported by the production configuration.")
+        if (self.TESTING):
+            raise ValueError("Testing is not supported by the production configuration.")
+        if (not self.DATABASE_URL):
+            raise ValueError("DATABASE_URL must be specified as an environment variable.")
+
+    def validate_secret_key(self, key):
```

Comment:
The following string was created using Flask's example of generating a [secret key](https://flask.palletsprojects.com/en/2.2.x/config/#SECRET_KEY). It has an entropy of 290bits and is estimated to take [centuries](https://lowe.github.io/tryzxcvbn/) for a computer to guess.
Yet, it doesn't contain uppercase letters nor does it use all the letters of the English alphabet. If you want to have some fun and have a better method for checking strong passwords or keys, then take a look at the [zxcvbn](https://github.com/dropbox/zxcvbn) https://github.com/dwolfhub/zxcvbn-python.

```
192b9bdd22ab9ed4d12e236c78afcb9a393ec15f71bbf5dc987d54727823bcbf
```

Reply:
Thanks for pointing out. Yeah, I almost removed it twice since I knew they were pretty weak checks. 

I removed the detailed checks and just required a key of at least 32 characters in [9b6eb48](https://github.com/hackforla/HomeUniteUs/pull/605/commits/9b6eb4806fdc4ac4e0321275a0c833f8a0ed98d3).